src/roiextractors/example_datasets/toy_example.py

```diff
@@ -87,7 +124,7 @@ def toy_example(
     size_y: int
         Size of y dimension (pixels)
     roi_size: int
-        Siz of ROI in x and y dimension (pixels)
+        Size of ROI in x and y dimension (pixels)
```

Comment:
@pauladkisson Could you add Codespell to the CI/pre-commit in a follow-up PR?

Example on NeuroConv: https://github.com/catalystneuro/neuroconv/blob/main/.github/workflows/codespell.yml

Reply:
See #236